examples/react/index.js

```diff
@@ -7,7 +7,9 @@ const selfClosing = <div />
 const multiLine = (
     <div>
         <span>Hello World</span>
-        <div>{selfClosing}</div>
+        <div>
```

Comment:
This will add more lines to the files where is the advantage

Reply:
This was done by prettier. Prettier is opinionated and don't give much options not sure if there is a way to disable this one.

Reply:
I updated prettier and it seems to fix this issue. it's now enforcing this to be in one line.